bundles/AdminBundle/Resources/public/js/pimcore/asset/video.js

```diff
@@ -338,10 +338,10 @@ pimcore.asset.video = Class.create(pimcore.asset.asset, {
         var previewContainerId = 'pimcore_video_preview_vr_' + this.id;
         this.previewPanel.update('<div id="' + previewContainerId + '" class="pimcore_asset_image_preview"></div>');
         var vrView = new VRView.Player('#' + previewContainerId, {
-            video: this.data['videoInfo']['previewUrl'],
+            video: '/admin/asset/serve-video-preview?id=' + this.data.id,
```

Comment:
```suggestion
            video: Routing.generate('pimcore_admin_asset_servevideopreview', {id: this.data.id}),
```
We should use proper route generation here.

Reply:
Indeed. And I've changed the size and aspect ratio of the preview to the more common 16:9.